hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/OzoneRepair.java

```diff
@@ -61,4 +71,33 @@ public OzoneConfiguration getOzoneConf() {
   public static void main(String[] argv) throws Exception {
     new OzoneRepair().run(argv);
   }
+
+  @Override
+  public int execute(String[] argv) {
+    String currentUser = getSystemUserName();
+    boolean shouldProceed = true;
+    if (!currentUser.equals(OZONE_DEFAULT_USER)) {
+      String s = getConsoleReadLineWithFormat(currentUser, OZONE_DEFAULT_USER);
+      shouldProceed = Boolean.parseBoolean(s) || "y".equalsIgnoreCase(s);
+    }
+    if (!shouldProceed) {
+      System.out.println("Aborting command.");
+      return 1;
+    }
+    System.out.println("Run as user: " + currentUser);
+
+    TracingUtil.initTracing("shell", createOzoneConfiguration());
+    String spanName = "ozone repair " + String.join(" ", argv);
+    return TracingUtil.executeInNewSpan(spanName,
+        () -> super.execute(argv));
```

Comment:
There's no network calls going out as part of repair commands. They run locally when nodes are stopped, so we don't need request tracing here like have for other commands.

Reply:
Yeah makes sense! removed tracing.